Starting backup...
Dumping database laravel...
Backup failed because: The dump process failed with a none successful exitcode.
Exitcode
========
2: Misuse of shell builtins

Output
======
<no output>

Error Output
============
mysqldump: Got error: 2002: "Can't connect to MySQL server on '127.0.0.1' (10061)" when trying to connect
.
#0 C:\xampp\htdocs\proyecto_dasce\vendor\spatie\db-dumper\src\DbDumper.php(200): Spatie\DbDumper\Exceptions\DumpFailed::processDidNotEndSuccessfully(Object(Symfony\Component\Process\Process))
#1 C:\xampp\htdocs\proyecto_dasce\vendor\spatie\db-dumper\src\Databases\MySql.php(171): Spatie\DbDumper\DbDumper->checkIfDumpWasSuccessFul(Object(Symfony\Component\Process\Process), 'C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\proyecto_dasce\vendor\spatie\laravel-backup\src\Tasks\Backup\BackupJob.php(296): Spatie\DbDumper\Databases\MySql->dumpToFile('C:\\xampp\\htdocs...')
#3 [internal function]: Spatie\Backup\Tasks\Backup\BackupJob->Spatie\Backup\Tasks\Backup\{closure}(Object(Spatie\DbDumper\Databases\MySql), 'mysql')
#4 C:\xampp\htdocs\proyecto_dasce\vendor\laravel\framework\src\Illuminate\Collections\Arr.php(749): array_map(Object(Closure), Array, Array)
#5 C:\xampp\htdocs\proyecto_dasce\vendor\laravel\framework\src\Illuminate\Collections\Collection.php(805): Illuminate\Support\Arr::map(Array, Object(Closure))
#6 C:\xampp\htdocs\proyecto_dasce\vendor\spatie\laravel-backup\src\Tasks\Backup\BackupJob.php(260): Illuminate\Support\Collection->map(Object(Closure))
#7 C:\xampp\htdocs\proyecto_dasce\vendor\spatie\laravel-backup\src\Tasks\Backup\BackupJob.php(199): Spatie\Backup\Tasks\Backup\BackupJob->dumpDatabases()
#8 C:\xampp\htdocs\proyecto_dasce\vendor\spatie\laravel-backup\src\Tasks\Backup\BackupJob.php(173): Spatie\Backup\Tasks\Backup\BackupJob->createBackupManifest()
#9 C:\xampp\htdocs\proyecto_dasce\vendor\spatie\laravel-backup\src\Commands\BackupCommand.php(72): Spatie\Backup\Tasks\Backup\BackupJob->run()
#10 C:\xampp\htdocs\proyecto_dasce\vendor\laravel\framework\src\Illuminate\Container\BoundMethod.php(36): Spatie\Backup\Commands\BackupCommand->handle()
#11 C:\xampp\htdocs\proyecto_dasce\vendor\laravel\framework\src\Illuminate\Container\Util.php(43): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#12 C:\xampp\htdocs\proyecto_dasce\vendor\laravel\framework\src\Illuminate\Container\BoundMethod.php(96): Illuminate\Container\Util::unwrapIfClosure(Object(Closure))
#13 C:\xampp\htdocs\proyecto_dasce\vendor\laravel\framework\src\Illuminate\Container\BoundMethod.php(35): Illuminate\Container\BoundMethod::callBoundMethod(Object(Illuminate\Foundation\Application), Array, Object(Closure))
#14 C:\xampp\htdocs\proyecto_dasce\vendor\laravel\framework\src\Illuminate\Container\Container.php(754): Illuminate\Container\BoundMethod::call(Object(Illuminate\Foundation\Application), Array, Array, NULL)
#15 C:\xampp\htdocs\proyecto_dasce\vendor\laravel\framework\src\Illuminate\Console\Command.php(211): Illuminate\Container\Container->call(Array)
#16 C:\xampp\htdocs\proyecto_dasce\vendor\symfony\console\Command\Command.php(318): Illuminate\Console\Command->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Illuminate\Console\OutputStyle))
#17 C:\xampp\htdocs\proyecto_dasce\vendor\laravel\framework\src\Illuminate\Console\Command.php(180): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Illuminate\Console\OutputStyle))
#18 C:\xampp\htdocs\proyecto_dasce\vendor\spatie\laravel-backup\src\Commands\BaseCommand.php(29): Illuminate\Console\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#19 C:\xampp\htdocs\proyecto_dasce\vendor\symfony\console\Application.php(1092): Spatie\Backup\Commands\BaseCommand->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#20 C:\xampp\htdocs\proyecto_dasce\vendor\symfony\console\Application.php(341): Symfony\Component\Console\Application->doRunCommand(Object(Spatie\Backup\Commands\BackupCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#21 C:\xampp\htdocs\proyecto_dasce\vendor\symfony\console\Application.php(192): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#22 C:\xampp\htdocs\proyecto_dasce\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(197): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#23 C:\xampp\htdocs\proyecto_dasce\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(1234): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#24 C:\xampp\htdocs\proyecto_dasce\artisan(16): Illuminate\Foundation\Application->handleCommand(Object(Symfony\Component\Console\Input\ArgvInput))
#25 {main}
Backup failed because: The dump process failed with a none successful exitcode.
Exitcode
========
2: Misuse of shell builtins

Output
======
<no output>

Error Output
============
mysqldump: Got error: 2002: "Can't connect to MySQL server on '127.0.0.1' (10061)" when trying to connect
.

   Symfony\Component\Mailer\Exception\TransportException 

  Connection could not be established with host "sandbox.smtp.mailtrap.io:2525": stream_socket_client(): php_network_getaddresses: getaddrinfo for sandbox.smtp.mailtrap.io failed: Host desconocido.

  at vendor\symfony\mailer\Transport\Smtp\Stream\SocketStream.php:154
    150▕         $streamContext = stream_context_create($options);
    151▕ 
    152▕         $timeout = $this->getTimeout();
    153▕         set_error_handler(function ($type, $msg) {
  ➜ 154▕             throw new TransportException(\sprintf('Connection could not be established with host "%s": ', $this->url).$msg);
    155▕         });
    156▕         try {
    157▕             $this->stream = stream_socket_client($this->url, $errno, $errstr, $timeout, \STREAM_CLIENT_CONNECT, $streamContext);
    158▕         } finally {

  1   [internal]:0
      Symfony\Component\Mailer\Transport\Smtp\Stream\SocketStream::Symfony\Component\Mailer\Transport\Smtp\Stream\{closure}("stream_socket_client(): php_network_getaddresses: getaddrinfo for sandbox.smtp.mailtrap.io failed: Host desconocido. ", "C:\xampp\htdocs\proyecto_dasce\vendor\symfony\mailer\Transport\Smtp\Stream\SocketStream.php")
  2   vendor\symfony\mailer\Transport\Smtp\Stream\SocketStream.php:157

